lib/matplotlib/tests/test_ft2font.py

```diff
@@ -76,3 +76,30 @@ def test_font_fallback_chinese(fig_test, fig_ref, family_name, file_name):
     ):
         fig_ref.text(0.05, .85 - 0.15*j, txt, family=ref_font)
         fig_test.text(0.05, .85 - 0.15*j, txt, family=test_font)
+
+
+@pytest.mark.parametrize(
+    "family_name, file_name",
+    [
+        ("WenQuanYi Zen Hei", "wqy-zenhei.ttc"),
+        ("Noto Sans CJK JP", "NotoSansCJK-Regular.ttc"),
```

Comment:
Not sure I fully get this (probably not), but for OSX the font seems to be `NotoSansCJKsc-Regular.otf`. Not that it may be required to add that...